common/core/web/keyboard-processor/README.md

```diff
@@ -0,0 +1,21 @@
+# Keyman:  Keyboard Processor Module
+The Original Code is (C) 2017-2020 SIL International
+
+The Keyboard Processor module is an internal component of KeymanWeb, seen within this repo at /web/.
+
+## Minimum Compilation Requirements
+
+* A local installation of [Node.js](https://nodejs.org/) v8.9+.
+	* Builds will call `npm install` to automatically install further necessary build dependencies.
+
+	* Linux users can run the following to update to LTS version of nodejs
```

Comment:
I'm not sure this platform, distro and version-dependent information that will soon be out of date belongs here.

Reply:
This directly matches existing text in [KeymanWeb's README.md](https://github.com/keymanapp/keyman/blob/master/web/README.md).

Since this and KeymanWeb should have the same build requirements (though possibly looser, for the keyboard-processor) I copied them here.